en-US/D.xml

```diff
@@ -87,7 +87,7 @@
 			<term>data store, datastore</term>
 			 <listitem>
 				<para>
-					<emphasis>n.</emphasis> Use the two-word form.
+					<emphasis>n.</emphasis> In most cases, write as two words. In a VMware context, use "datastore".
 				</para>
```

Comment:
What do you mean by "most cases"? Is VMware usage the only exception? If not, we probably need to identify what the other cases are.

Reply:
Updated